packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/BlocksInput/BlocksToolbar.tsx

```diff
@@ -341,10 +344,12 @@ const isListNode = (node: unknown): node is Block<'list'> => {
 interface ListButtonProps {
   block: BlocksStore['list-ordered'] | BlocksStore['list-unordered'];
   format: Block<'list'>['format'];
+  location?: 'toolbar' | 'menu';
 }
 
-const ListButton = ({ block, format }: ListButtonProps) => {
+const ListButton = ({ block, format, location = 'toolbar' }: ListButtonProps) => {
```

Comment:
I'm really not a fan of this 'location' pattern, but it's a quick way to adapt the UI while benefiting from the existing logic. But the logic should be moved into the blocks store in my opinion, just like for the modifiers.